src/stories/HdAlert.stories.js

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { sanitize } from 'dompurify';
+import { storiesOf } from '@storybook/vue';
+import {
+  text,
+  boolean,
+  select,
+} from '@storybook/addon-knobs';
+import HdAlert from 'hd-blocks/components/HdAlert.vue';
+
+storiesOf('HdAlert', module)
+  .add('Info (default)', () => ({
+    components: { HdAlert },
+    template: `
+      <HdAlert
+        message="Karaoke means &#8220;empty orchestra&#8221; in Japanese."
+      />
+    `,
+  }))
+  .add('Success', () => ({
+    components: { HdAlert },
+    template: `
+      <HdAlert
+        type="success"
+        message="A success message"
+      />
+    `,
+  }))
+  .add('Warning', () => ({
+    components: { HdAlert },
+    template: `
+      <HdAlert
+        type="warning"
+        message="A warning message"
+      />
+    `,
+  }))
+  .add('Error', () => ({
+    components: { HdAlert },
+    template: `
+      <HdAlert
+        type="error"
+        message="An error message"
+      />
+    `,
+  }))
+  .add('Playground 🎛📝', () => ({
+    components: { HdAlert },
+    props: {
+      type: {
+        type: String,
+        default: select('type', ['', 'success', 'info', 'warning', 'error']),
```

Comment:
To avoid it (if we add more options), I would create the types in a different file so they can be easily imported among other components/files.

Reply:
1- That makes sense, thanks for the suggestion! Please check the new commit. I also went with dynamic stories, so we don't have to add/remove a stories if we change the types.
2- That's true 😅 I removed that option.